_docs/policies/tests.md

```diff
@@ -0,0 +1,47 @@
+# `status-go` Test Policy
+
+- [Creating Tests](#creating-tests)
+- [Flaky Tests](#flaky-tests)
+
+## Creating Tests
+
+- All new functionality MUST be introduced with tests that:
+  - Prove that the functionality performs as described
+  - Can be falsified
+  - Are resistant to fuzzing
+- All new tests MUST BE validated via 1000 local tests. Ensuring that the test runs and passes consistently every time gives confidence that the test is not flaky.
+    - add the `-count` flag to the test command eg: `-count 1000`
+    - OR wrap your test in a for loop
+        ```go
+        func TestTheThing(t *testing.T) {
+            for i := 1; i < 1000; i++ {
+                fmt.Println("Test Run", i)
+                // your test body goes in here
+            }
+        }
+        ```
+## Flaky Tests
+
+- All flaky tests / failing tests must be resolved
+  - Do not introduce flaky tests to the codebase
+- Steps to resolving flaky test
+  - Ensure that the test is definitely flaky and that it wasn’t you that introduced a bug that made the test flaky.
+      - Is a new test you’ve written flaky?
+        - Ok, you need to fix that before merge is acceptable.
```

Comment:
Seems to be duplication of `Do not introduce flaky tests to the codebase` (I would remove the former).

Reply:
Yeah, this language is way too informal for a policy. I will change it.